dockerfile

```
@@ -33,8 +34,9 @@ RUN apt-get update && \
         libxcb-shape0 \
         && apt-get clean
 
-# install napari release version
-RUN pip3 install napari[all]
+# install napari from repo
+# see https://github.com/pypa/pip/issues/6548#issuecomment-498615461 for syntax
+RUN pip3 install "napari[all] @ git+https://github.com/napari/napari.git@${NAPARI_COMMIT}"
```

Maybe we can also install from the already checked out repo. We just need to pass it in the context (if not already passed). In other words:

```suggestion
RUN pip3 install ".[all]"
```

---

it will not work until we copy the repo content to the docker container.

---

Then let's leave it like this. I think it's simpler.